src/rust/ensogl/src/system/gpu/data/texture/class.rs

```diff
@@ -45,19 +45,65 @@ impl Drop for TextureBindGuard {
 
 
 
+// ==================
+// === Parameters ===
+// ==================
+
+/// Parameters that need to be set when binding a texture.
```

Comment:
Please provide link to MDN or something explaining the concepts better.

Reply:
Moreover, please write more in the docs that these parameters are about filtering etc. When reading the source code it should be more easy to follow what it is about.